coriolisclient/cli/endpoints.py

```diff
@@ -108,6 +112,13 @@ def take_action(self, args):
             conn_info,
             args.description)
 
+        if not args.skip_validation:
+            coriolis_endpoints = self.app.client_manager.coriolis.endpoints
```

Comment:
Try to avoid these types of "proxy variables" whenever possible:
```python3
if not args.skip_validation:
    valid, message = (
        self.app.client_manager.coriolis.endpoints.validate_connection(
            endpoint.id))
        ...
```